packages/core/helper-plugin/lib/src/components/RelationInput/components/RelationList.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+
+const ShadowBox = styled(Box)`
+  position: relative;
+
+  &:before,
+  &:after {
+    position: absolute;
+    opacity: 0.8;
+    width: 100%;
+    height: 4px;
+    z-index: 1;
+  }
+
+  &:before {
+    content: ${({ overflow }) =>
+      overflow === 'top-bottom' || overflow === 'top' ? "''" : undefined};
+    background: linear-gradient(#dfdfe7 0%, rgba(0, 0, 0, 0) 100%);
```

Comment:
I checked the design-system and it seems that this is using the same shadow colors as e.g. [`filterShadow`](https://github.com/strapi/design-system/blob/f45f61988eab512a17a8a520e15006272e7954a7/packages/strapi-design-system/src/themes/lightTheme/light-shadows.js#L3). What do you think about:

1. Adding a comment for that, so that we have a chance to refactor it at some point
2. Rewrite the HEX notion into RGBA, to be consistent with the design-system values. This would also allow you to remove the `opacity` attribute: `rgba(33, 33, 52, 0.1)`